Client/Application/AppDelegate.swift

```diff
@@ -20,10 +20,17 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
                                                 .rustSyncManagerComponent
                                                 .value()
                                                 .useRustSyncManager
+    private let creditCardAutofillStatus = FxNimbus.shared
```

Comment:
I added this here because I can't access the `FeatureFlaggable` protocol from `RustSyncManagerAPI.swift`.